src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -4378,4 +4378,4 @@ Debug.Arena = 0
 #                                                                                                 #
 # GAME SETTINGS END                                                                               #
 #                                                                                                 #
-###################################################################################################
+###################################################################################################
```

Comment:
Newline removed, I believe unintentionally.

Reply:
Ye I saw it too, not sure how it got removed but didnt think it would matter 😅 Anyway, reverted it.